### DISCUSSION TITLE
mkdocs is included in this cookiecutter template

### DISCUSSION THREAD

**Discussion by zillionare** [mkdocs/mkdocs#2411] [Show and tell]
I'm huge fan of mkdocs/markdown, and made a cookiecutter template in which mkdocs is the documentation builder. You may consider this template (and also a standalone tool called "python project wizard") if you'd like to use mkdocs too.

The template is [here](https://github.com/zillionare/cookiecutter-pypackage)

**Comment by Andre601** [comment]
I myself offer [Andre601/mkdocs-template](https://github.com/Andre601/mkdocs-template) for easy creation of docs using GitHub pages, Mkdocs and Material for Mkdocs.

**Comment by pawamoy** [comment]
And here are mine :stuck_out_tongue: 

- https://github.com/pawamoy/copier-pdm
- https://github.com/pawamoy/copier-poetry
